src/main/java/org/jabref/gui/menus/ChangeEntryTypeMenu.java

```diff
@@ -92,4 +102,116 @@ private void populate(ObservableList<MenuItem> items, Collection<BibEntryType> t
+                case InReference -> {
+                    return Localization.lang("An article in a work of reference. This is a more specific variant of the generic \"InCollection\" entry type.");
+                }
+                case MvBook -> {
+                    return Localization.lang("A multi-volume \"Book\".");
+                }
+                case MvCollection -> {
+                    return Localization.lang("A multi-volume \"Collection\".");
+                }
+                case MvProceedings -> {
+                    return Localization.lang("A multi-volume \"Proceedings\" entry.");
+                }
+                case MvReference -> {
+                    return Localization.lang("A multi-volume \"Reference\" entry. The standard styles will treat this entry type as an alias for \"MvCollection\".");
+                }
+                case Online -> {
+                    return Localization.lang("This entry type is intended for sources such as web sites which are intrinsically online resources.");
+                }
+                case Reference -> {
+                    return Localization.lang("A single-volume work of reference such as an encyclopedia or a dictionary.");
+                }
+                case Report -> {
+                    return Localization.lang("A technical report, research report, or white paper published by a university or some other institution.");
+                }
+                case Set -> {
+                    return Localization.lang("An entry set is a group of entries which are cited as a single reference and listed as a single item in the bibliography.");
+                }
+                case SuppBook -> {
+                    return Localization.lang("Supplemental material in a \"Book\". This type is provided for elements such as prefaces, introductions, forewords, afterwords, etc. which often have a generic title only.");
+                }
+                case SuppCollection -> {
+                    return Localization.lang("Supplemental material in a \"Collection\".");
+                }
+                case SuppPeriodical -> {
+                    return Localization.lang("Supplemental material in a \"Periodical\". This type may be useful when referring to items such as regular columns, obituaries, letters to the editor, etc. which only have a generic title.");
+                }
+                case Thesis -> {
+                    return Localization.lang("A thesis written for an educational institution to satisfy the requirements for a degree.");
+                }
+                case WWW -> {
+                    return Localization.lang("An alias for \"Online\", provided for jurabib compatibility.");
+                }
+                case Software -> {
+                    return Localization.lang("Computer software. The standard styles will treat this entry type as an alias for \"Misc\".");
+                }
+                case DATESET -> {
```

Comment:
It would be cool if could do a global renaming (refactor/rename) to change the type to Dataset

Reply:
@Siedlerchr do you want it to be Dataset or DataSet?

Reply:
Dataset

Reply:
@Siedlerchr what do you think about my latest commit?

Reply:
LGTM now! I just merged in master and resolved the conflicts. When all relevant tests pass we can merge.